internal/namespaces/object/v1/custom_bucket.go

```diff
@@ -0,0 +1,559 @@
+package object
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	"github.com/scaleway/scaleway-cli/v2/internal/human"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+type bucketInfo struct {
+	ID               string
+	Region           scw.Region
+	APIEndpoint      string
+	BucketEndpoint   string
+	EnableVersioning bool
+	Tags             []types.Tag
+	ACL              []CustomS3ACLGrant
+	Owner            string
+}
+
+func bucketInfoMarshalerFunc(i interface{}, opt *human.MarshalOpt) (string, error) {
+	// To avoid recursion of human.Marshal we create a dummy type
+	type tmp bucketInfo
+	info := tmp(i.(bucketInfo))
+
+	opt.Sections = []*human.MarshalSection{
+		{
+			FieldName:   "Tags",
+			HideIfEmpty: true,
+		},
+		{
+			FieldName:   "ACL",
+			HideIfEmpty: true,
+		},
+	}
+	str, err := human.Marshal(info, opt)
+	if err != nil {
+		return "", err
+	}
+	return str, nil
+}
+
+type BucketResponse struct {
+	SuccessResult *core.SuccessResult
+	BucketInfo    bucketInfo
```

Comment:
Let's use pointer as default instead of a mix
```suggestion
	SuccessResult *core.SuccessResult
	BucketInfo    *bucketInfo
```